File: VariablesYOperaciones/C#/VelocidadDeLaLuz.cs
using System;

namespace VelocidadDeLaLuz
{
		public class Program
		{
			public static void Main (string[] args)
			{
				int Seg_Recorridos,Distancia_Recorrida;
				Console.WriteLine("Ingrese los segundos: ");
							Seg_Recorridos=Convert.ToInt32(Console.ReadLine());
					Distancia_Recorrida=Seg_Recorridos*300000;
				Console.WriteLine("La distancia Recorrida es de: "+Distancia_Recorrida);
			}
		}
}

File: VariablesYOperaciones/VisualBasic/VelocidadDeLaLuz.vb
Imports System

Module Program
    Sub Main(args As String())
        Dim segundos, R As Integer
        Dim distancia As ULong = 300000

        Console.Write("Ingrese el valor del tiempo en segundos: ")
        segundos = Console.ReadLine()

        R = distancia * segundos
        Console.WriteLine("La distancia recorrida en " & segundos & " segundos es de: " & R & " kilómetros")
    End Sub
End Module